solr/core/src/java/org/apache/solr/handler/SolrConfigHandler.java

```diff
@@ -104,9 +105,6 @@
 public class SolrConfigHandler extends RequestHandlerBase
     implements SolrCoreAware, PermissionNameProvider {
   private static final Logger log = LoggerFactory.getLogger(MethodHandles.lookup().lookupClass());
-  public static final String CONFIGSET_EDITING_DISABLED_ARG = "disable.configEdit";
```

Comment:
@janhoy if you didn't notice, we're elevating this very internal undocumented sys prop to node level to increase the scope beyond config editing of a core live to also include disabling cluster config edit.  Obviously the name of this sys prop is poor and your new efforts to standardize suggest we should take the opportunity to have a nicer name?

Reply:
Sorry for the back & forth here... I had thought let's use `disable.configEdit` but I don't want to shine a light (document) a badly named config, especially at a time when a config naming standardization effort is underway.

Absent further feedback, I think we should use a **new** property, like `solr.cluster.plugin.mutable` using `EnvUtils`.  It's scoped to just the `/cluster/plugin` API and thus doesn't have a debatable scope (too broad, like all config editing, which should get more agreement/care).  Such a narrow scope also suggests moving the setting to ContainerPluginsRegistry or ZkClusterPluginsSource.